presto-product-tests/conf/presto/etc/fileauth.properties

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
same comments like for presto-product-tests/conf/presto/etc/file_auth. See https://github.com/prestosql/presto/issues/1766

Reply:
@kokosing If I follow #1766 here will it affect the existing tests?

Reply:
What do you mean? In general #1766 should not affect tests, but it is more about tests infra (file locations etc)